tests/test_dataclass.py

```diff
@@ -400,6 +401,7 @@ class PyType:
 
 
 def test_dataclass_repeated_enum_field():
+    @register_aliases(aliases=["test_dataclass.OldEnum"])
```

Comment:
this is cool 👌 having the fully qualified name here allows quite a lot of freedom in moving things around!